packages/server/graphql/public/typeDefs/Organization.graphql

```diff
@@ -72,6 +72,21 @@ type Organization {
   """
   periodStart: DateTime
 
+  """
+  Flag the organization as exceeding the tariff limits by setting a datetime
+  """
+  tierLimitExceededAt: DateTime
+
+  """
+  Schedule the organization to be locked at
+  """
+  scheduledLockAt: DateTime
+
+  """
+  Organization locked at
+  """
+  lockedAt: DateTime
```

Comment:
Just wondering why we need both `lockedAt` and `scheduledLockAt`?

Reply:
@nickoferrall We plan have a cron that will actually lock an organization, lock all related teams and send an email. This is a flag that locking process is finished, emails were sent etc.